### ISSUE TITLE
New category of Additions?

### ISSUE THREAD

**Issue by DariaElagina** [BetaMasaheft/Documentation#1765]
Dear all,

just as a proposal, I would like to discuss the possibility of adding a new category Paratext to the additions. I have noticed, that it is not uncommon that titles, indexes or other paratextual elements are added later to the manuscript. I do not see any fitting category for such additions so far. Maybe there is already a way to encode it, I am not aware about... 

For example, index on fol. 3v: http://www.bl.uk/manuscripts/Viewer.aspx?ref=or_732_fs001r

Any better ideas? @DenisNosnitsin1970  @thea-m  @eu-genia

**Comment by thea-m** [comment]
No, this category doesn't exist yet. I think that under the current guidelines these would mostly be encoded as extras. I agree that such a category for additions would make sense, however I think that it would be quite difficult in practice to distinguish between such elements that are truly additions and others that were included in the manuscript in one act of production with the main text and would then be encoded either as msItems or extras. In the example given here, I could not make a choice, but would tend towards not interpreting it as a later addition. 
Nevertheless I'm sure that such paratextual elements might be added as additions and that the category could be created, but then should be used very carefully. Let's see what everyone else says.

**Comment by eu-genia** [comment]
The example does seem to be a separate production layer to me (a somewhat different, though similar, hand, placed on a blanco verso before the main text - seems the same hand/layer as the list/inventory on f. 366v after the text), so in this case I would tend to see 3v as an added text, though clearly related to the main one, and possibly added very close to the time of production. So yes, very difficult to decide between an addition proper and an extra.
Still, whether an addition or extra, a type could be helpful. 
Another question: is "paratext" the right term? It might be a bit broader than what we need here (but of course we can have our own definition)

**Comment by DenisNosnitsin1970** [comment]
"Paratext theory" was recently proposed by a concrete scholar, I do not know, can we redefine what he has defined? It is a little elusive as it is a category coming from literary studies and requiring a point of view different from that of cataloguing (is it used in cataloguing?). In my opinion, it is just a category that is not on the same level if compared with "Additions" and "Extras".  To sum up, I think "paratextual" can be seen as an additional qualification for "Additions" and "Extras", or even for parts of the main work (possibly further elements?) but rather not as a substitute for either of them or a new category.   Another thing - it would be useful to collect more examples to get a better idea thereof.

**Comment by eu-genia** [comment]
Exactly, in literary studies, paratext is roughly everything that is not the main text but still belongs to the book (so the impressum/colophon, exlibris/note of possession/donation, scribal explicit - all these are paratexts in book studies, and we have different ways for encoding these). The question is, how to define an "addition", which is added later by a secondary hand, but is not a guest text or document etc. but something enhancing the main text, if we need to have a type for it. This is a difficult question though, sometimes getting into the mind of the person who was adding and deciding, was he adding something because he thought the texts belong together, or just because he saw the space- Many of what we currently encode as additions with "guest text" were also seen by their scribes as enhancing the main text, e.g. take a hymn to the saint whose vita is in the ms,

**Comment by DariaElagina** [comment]
Maybe Peritext could be a more appropriate term? If it would be just used as an additional category for additiones, it would still be distinguishable from other elements which do not fall within the category of additions (colophon, etc.)

**Comment by DenisNosnitsin1970** [comment]
What do you mean by "additional category for additions"? Additiones + Peritexts + Extras?

**Comment by eu-genia** [comment]
I guess Daria means a new type for additions (or extras), in addition to GuestText, DonationNote etc.

**Comment by DariaElagina** [comment]
Yes, thank you Zhenia! I mean the type, sorry for confusion.

**Comment by eu-genia** [comment]
addition `@type="peritext"` for indices/tables of contents/notes to the text/titles could make sense IMHO but I am a bit afraid of overlapping definitions

**Comment by DenisNosnitsin1970** [comment]
What will be marked, for instance, as peritext?

**Comment by DenisNosnitsin1970** [comment]
Tables of contens and indices are also part-works, "peritext" is an additional attribute or not?

**Comment by eu-genia** [comment]
if they are added, not by the main scribe but at another layer, they are not msItems
of course we are talking of ADDED indices/tables of contents/notes to the text/titles

**Comment by DenisNosnitsin1970** [comment]
For me, it is a bit abstract until now. Can some examples be produced to show what will be what and how it will work? (with reference to  several concrete manuscripts that can be viewed).  The example above is not very representative (indeed, why it is an addition and not msItem written in a different hand?). Also I see it a bit problematic that some (additional) writings will be marked as peritexts, but some that we usually encode as part-works (e.g. prefatory parts to Gospels, halelluyatic tables) will be not marked as such, even though there "paratextual function" is strongly pronounced.

**Comment by DariaElagina** [comment]
For example, here: http://www.bl.uk/manuscripts/Viewer.aspx?ref=or_741_f001r  fol. 150r
There is a colophon for the section written in a later hand. I think, it might have been added to be able to insert the names of the new owners. I have encoded it as an addition, but there is no type that can be assigned to it. I do not think it would be correct to consider it as a proper colophon, since it is clearly a later layer of production.

**Comment by eu-genia** [comment]
Add this to the set of issues dealing with additions and extras to close https://github.com/BetaMasaheft/Documentation/issues/1919
